qml/WebView.qml

```diff
@@ -30,7 +31,8 @@ WebView {
     property string webViewBackgroundSource: "images/background-startpage.png"
     property string webViewPlaceholderSource: "images/startpage-placeholder.png"
     id: webViewItem
-    anchors.top: navigationBar.alwaysShowProgressBar?progressBar.bottom:navigationBar.bottom
+    anchors.top: navigationBar.alwaysShowProgressBar?
+    progressBar.bottom:navigationBar.bottom
```

Comment:
This two lines are really misleading, if we don't see the "?" at the end of line 34